db.js

```diff
@@ -116,6 +117,31 @@ if (couchUrl) {
   };
 
   module.exports.sanitizeResponse = sanitizeResponse;
+
+  const runBatch = (query, iteratee, batchSize, skip, callback) => {
+    query(skip, (err, response) => {
+      if (err) {
+        return callback(err);
+      }
+      console.log(`        Processing ${skip} to ${skip + batchSize} docs of ${response.total_rows} total`);
+      iteratee(response.rows, err => {
+        const keepGoing = response.total_rows > (skip + batchSize);
+        callback(err, keepGoing);
+      });
+    });
+  };
+
+  module.exports.batch = (query, iteratee, batchSize, callback) => {
```

Comment:
It's also worth considering that if we're having a technique elevated into the "standard library" (as it were), we probably want to do skipping the more correct / performant way, using keys, as opposed to numeric record skipping.